sync/syncer.go

```diff
@@ -172,12 +173,16 @@ func (s *Syncer) Run(ctx context.Context) error {
 				"phase", phaseLabel[doFetch],
 				"more", resp.MoreToFollow,
 				"cursor", resp.Cursor,
-				"devices", len(resp.Devices),
+				// "device-events" because there could be multiple events for the same device.
```

Comment:
I see the point here, I do. However `devices` is just the name of the key that Apple gives us. It doesn't necessarily imply _unique_ devices.